src/solver/application.cpp

```diff
@@ -493,8 +493,10 @@ void Application::readDataForTheStudy(Data::StudyLoadOptions& options)
 
 void Application::saveElapsedTime()
 {
-    pTotalTimer.stop();
+    if (!pStudy)
```

Comment:
ok. So it makes (private) data member **shouldExecute** redundant, and therefore useless, right ?

Reply:
Done ! It works